runtime/protocol/http-client/common/src/aws/smithy/kotlin/runtime/http/operation/SdkOperationExecution.kt

```diff
@@ -275,10 +275,13 @@ internal class AuthHandler<Input, Output>(
             "auth.scheme_id" to authScheme.schemeId.id
         }
 
+        // signing properties need to propagate from AuthOption to signer
```

Comment:
**Nit**: This comment only tells half the story now. We should update it to something like "properties need to propagate from AuthOption to signer and identity provider".